download-data.py

```diff
@@ -4,20 +4,150 @@
 
 import argparse
 import datetime
+import fnmatch
 import os
 import pathlib
 import re
 import subprocess
 import sys
-from typing import List
+from typing import List, Set
 
 # enable text formatting on Windows
 os.system("")
 
 RELEASE_BUCKET = "openscpca-data-release"
+RESULTS_BUCKET = "openscpca-nf-workflow-results"
 TEST_BUCKET = "openscpca-temp-simdata"  # TODO: change to correct bucket
 
 
+def get_releases(bucket: str, profile: str) -> List[str]:
+    """
+    Get the list of available releases from the OpenScPCA data bucket.
+    """
+    ls_cmd = ["aws", "s3", "ls", f"s3://{bucket}/"]
+    if profile:
+        ls_cmd += ["--profile", profile]
+    if bucket == TEST_BUCKET:
+        ls_cmd += ["--no-sign-request"]
+    ls_result = subprocess.run(ls_cmd, capture_output=True, text=True)
+    if ls_result.returncode:
+        print(
+            "Error listing release versions from the OpenScPCA release bucket.",
+            " Ensure you have the correct AWS permissions to access OpenScPCA data.",
```

Comment:
Just because I've seen this error message myself, can we update it to remind people to login or use the correct profile since that might be why they don't have access 😬